### PR TITLE
SAN-28: Update Pay Penalty Page

### PR DESCRIPTION
**Pull request by klyttle-ch**
Update Pay Penalty Page

### CODE REVIEW

File: src/main/resources/templates/pps/home.html [hunk 1]
```diff
@@ -14,105 +14,97 @@
...
+                    </svg>
+                </a>
+
                 <p class="govuk-body">
-                    You can
-                    <a class="govuk-link govuk-link--no-visited-state" id="appeals-link"
-                       data-event-id="lfp - appeal a penalty link" href="/appeal-a-penalty/">
-                        appeal a late filing penalty</a>
-                    , but you'll need to show that the reasons for filing late were exceptional.
+                    <a href="https://www.gov.uk/government/publications/companies-house-approach-to-financial-penalties"
```
Comment (dgordon-ch) [inline src/main/resources/templates/pps/home.html:61]: Might be job for separate ticket so can leave this full url in for now, but wondering if it's worth using env prop for the govUrl in the places where we use it -- similar to the churl we have.

File: src/main/resources/templates/pps/home.html [hunk 2]
```diff
@@ -14,105 +14,97 @@
...
+                            to the right team.
+                        </div>
+                    </details>
+                </div>
+
+                <p class="govuk-body">You must pay the full amount.</p>
+                <p class="govuk-body">It takes around 5 minutes to pay.</p>
+
+                <a href="what-does-the-penalty-reference-start-with" role="button" draggable="false"
+                   class="govuk-button govuk-button--start" data-module="govuk-button" data-event-id="pps - Start now"
```
Comment (dgordon-ch) [inline src/main/resources/templates/pps/home.html:51]: On the data-event-id, this stuff will have an overhaul as part of the Matomo configuration so we can look at this further at that point, will need to plumb in this sort of id tracking for more than just a single button too.

File: src/main/resources/templates/pps/home.html [hunk 3]
```diff
@@ -14,105 +14,97 @@
...
-                    , but you'll need to show that the reasons for filing late were exceptional.
+                    <a href="https://www.gov.uk/government/publications/companies-house-approach-to-financial-penalties"
+                       class="govuk-link">Read the guidance for more information about Companies House penalties</a>.
                 </p>
 
-                <p class="govuk-body">It takes around 5 minutes to pay.</p>
-                <div class="form-group">
-                    <input id="next-button" class="govuk-button" data-event-id="lfp - Start now"
-                           onclick="_paq.push(['trackGoal', 3]);" type="submit" role="button" value="Start now"/>
+                <div class="govuk-!-margin-top-6">
```
Comment (dgordon-ch) [inline src/main/resources/templates/pps/home.html:65]: Is this margin standard across other pages?

File: src/main/resources/templates/pps/home.html [hunk 4]
```diff
@@ -14,105 +14,97 @@
         <div class="govuk-grid-row">
             <div class="govuk-grid-column-two-thirds">
                 <h1 class="govuk-heading-xl" id="main-heading">
-                    Pay a penalty for filing your company accounts late
+                    Pay a penalty to Companies House
                 </h1>
-                <p class="govuk-body-l">Use this service to pay
-                    a company's late filing penalty.</p>
-
+                <p class="govuk-body-l">Use this service to make an online payment by credit or debit card if you've
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/home.html:19]: The class here might need to change to be `p class="govuk-body"`? - referring to the v9 prototype, see https://github.com/companieshouse/sanctions-payment-prototype/blob/main/app/views/v9/01-pay-a-penalty-to-companies-house.html#L46

Comment (klyttle-ch) [reply src/main/resources/templates/pps/home.html:19]: Going to go ahead and change this to "govuk-body" to follow prototype @rbull-ch

File: src/main/resources/templates/pps/home.html [hunk 5]
```diff
@@ -14,105 +14,97 @@
...
                 </ul>
 
+                <div class="govuk-!-margin-top-6">
+                    <details class="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text">I've lost my penalty notice
+                            </span>
+                        </summary>
+                        <div class="govuk-details__text">
+                            <p class="govuk-body"> Email <a href="mailto:enquiries@companieshouse.gov.uk"
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/home.html:36]: This should be a mailto link e.g. `a href="mailto:enquiries@companieshouse.gov.uk"`
See `src/main/resources/templates/pps/confirmationPage.html`.
Also worth noting that their appears to be an an application property that we might be able to use in our template - see enquiries in `src/main/resources/application.properties`

File: src/main/resources/templates/pps/home.html [hunk 6]
```diff
@@ -14,105 +14,97 @@
...
+                                                            class="govuk-link">enquiries@companieshouse.gov.uk</a> to
+                                let us know if you have lost your penalty notice. Use 'Lost penalty notice' as the email
+                                subject.
+                            </p>
+                            <p></p>Tell us as much information as you can about the penalty so we can pass your message
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/home.html:41]: Should the text here appear inside the paragraph tag?

File: src/main/resources/templates/pps/home.html [hunk 7]
```diff
@@ -14,105 +14,97 @@
...
+                            to the right team.
+                        </div>
+                    </details>
+                </div>
+
+                <p class="govuk-body">You must pay the full amount.</p>
+                <p class="govuk-body">It takes around 5 minutes to pay.</p>
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/home.html:48]: Should this text be `It usually takes around 5 minutes to pay. We'll email you a receipt.`
Looking at https://sanctions-payment-prototype-0d6d68d6c82d.herokuapp.com/v9/01-pay-a-penalty-to-companies-house#

File: src/main/resources/templates/pps/home.html (hunk 5 above)
Comment (klyttle-ch) [reply src/main/resources/templates/pps/home.html:36]: @rbull-ch please double check this as I have used a mailto link

File: src/main/resources/templates/pps/home.html [hunk 8]
```diff
@@ -14,105 +14,97 @@
...
                 </p>
 
-                <p class="govuk-body">It takes around 5 minutes to pay.</p>
-                <div class="form-group">
-                    <input id="next-button" class="govuk-button" data-event-id="lfp - Start now"
-                           onclick="_paq.push(['trackGoal', 3]);" type="submit" role="button" value="Start now"/>
+                <div class="govuk-!-margin-top-6">
+                    <details class="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text"> Pay by bank transfer</span>
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/home.html:68]: Can we remove the leading space here in this span tag?

File: src/main/resources/templates/pps/home.html [hunk 9]
```diff
@@ -14,105 +14,97 @@
...
+                    <details class="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text"> I want to appeal a penalty </span>
+                        </summary>
+                        <div class="govuk-details__text">
+                            <h3 class="govuk-heading-s">If you have a late filing penalty</h3>
+                            This option is only for penalties for late filing of accounts. You can <a
+                                href="https://www.gov.uk/appeal-a-penalty-for-filing-your-company-accounts-late"
+                                class="govuk-link">appeal a late filing penalty (for filing accounts late) through
+                            Companies House</a>. You must show that the reasons are exceptional.<p></p>
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/home.html:89]: Please can you check your paragraph tags throughout where we have `<p></p>`.
Should we be putting the paragraph text inside them?

File: src/main/resources/templates/pps/home.html (hunk 5 above)
Comment (rbull-ch) [reply src/main/resources/templates/pps/home.html:36]: Ah sorry - right you are @klyttle-ch 
Was reviewing the prototype for first time and got my wires crossed!

File: src/main/resources/templates/pps/home.html (hunk 7 above)
Comment (rbull-ch) [reply src/main/resources/templates/pps/home.html]: Can you check the first bit says usually like in `It usually takes around 5 minutes to pay. We'll email you a receipt.`?

File: src/main/resources/templates/pps/home.html [hunk 10]
```diff
@@ -14,105 +14,97 @@
...
+                    </details>
+                </div>
+
+                <p class="govuk-body">You must pay the full amount.</p>
+                <p class="govuk-body">It takes around 5 minutes to pay.</p>
+
+                <a href="what-does-the-penalty-reference-start-with" role="button" draggable="false"
+                   class="govuk-button govuk-button--start" data-module="govuk-button" data-event-id="pps - Start now"
+                   onclick="_paq.push(['trackGoal', 3]);">Start now
+                    <!--Waiting for SAN-29 to complete link above (line 50)-->
```
[the remark this answers is not in the document]
Comment (dgordon-ch) [reply src/main/resources/templates/pps/home.html:53]: Probably best to add comment into SAN-29 to update this as part of the changes to that ticket. Can remove this comment if we have it logged in JIra ticket.

File: src/main/resources/templates/pps/home.html [hunk 11]
```diff
@@ -71,7 +70,6 @@ <h1 class="govuk-heading-xl" id="main-heading">
                             <p class="govuk-body">This payment method usually takes longer to process (up to 4 working
                                 days). <a href="late-filing-penalty/bank-transfer/which-penalty-service" class="govuk-link">Find out which
                                     account details to use for a bank transfer</a>.
-                                <!--Waiting for SAN-71 to complete link above (line 72)--></p>
```
Comment (rbull-ch) [inline src/main/resources/templates/pps/home.html]: Can you check the paragraph end tag needs adding back - looks to be accidentally deleted?

**Review by rbull-ch — APPROVED** [review]

**Review by acallaghan-ch — APPROVED** [review]

**Review by rbull-ch — APPROVED** [review]
Approved - can this be merged please?

**Pull request merged**